bin/configurable-http-proxy

```diff
@@ -317,9 +317,9 @@ if (args.ip === "*") {
   args.ip = "";
 }
 listen.ip = args.ip;
-listen.apiIp = args.apiIp || "localhost";
+listen.apiIp = args.apiIp;
 listen.apiPort = args.apiPort || listen.port + 1;
-listen.metricsIp = args.metricsIp || "0.0.0.0";
+listen.metricsIp = args.metricsIp;
```

Comment:
Note that these were defaults that were set above during CLI parsing anyhow. By changing this, the actual change in logic outcomes are when we pass a blank string and `args.apiIp || "localhost"` no longer overrides the falsy blank string.